meta/runtime.yml

```diff
@@ -28,10 +28,3 @@ action_groups:
         - vm_list_group_by_clusters_info
         - vm_resource_info
         - vm_portgroup_info
-plugin_routing:
-    modules:
-        vm_list_group_by_clusters:
-            redirect: vmware.vmware.vm_list_group_by_clusters_info
-            deprecation:
-                removal_version: 2.0.0
-                warning_text: Use M(vmware.vmware.vm_list_group_by_clusters_info) instead.
```

Comment:
I think best practice is to [tombstone](https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#tombstoning-a-module-or-plugin-in-a-collection) the module.

I haven't done this myself yet, but it looks like you should remove the redirect and change deprecation to tombstone.